src/instance.rs

```diff
@@ -164,6 +168,20 @@ wrappable_struct!(
 
 class!(RubyExportedFunctions);
 
+#[rustfmt::skip]
+methods!(
+    RubyInstance,
```

Comment:
I think you want to declare the method on `RubyExportedFunctions`, not `RubyInstance`, am I correct?

Reply:
Yes. I'fix.